Stp.Data/TestingDbContext.cs

```diff
@@ -25,12 +25,25 @@ protected override void OnModelCreating(ModelBuilder modelBuilder)
             modelBuilder.Entity<TaskCategory>().HasQueryFilter(x => EF.Property<bool>(x, "IsDeleted") == false);
             modelBuilder.Entity<StpTask>().HasQueryFilter(x => EF.Property<bool>(x, "IsDeleted") == false);
             modelBuilder.Entity<MultichoiceTaskAnswer>().HasQueryFilter(x => EF.Property<bool>(x, "IsDeleted") == false);
+
+            modelBuilder.Entity<TaskAndSkill>()
+                .HasKey(ts => new { ts.TaskId, ts.SkillId });
+            modelBuilder.Entity<TaskAndSkill>()
+                .HasOne(t => t.Task)
+                .WithMany(ts => ts.TaskAndSkills)
+                .HasForeignKey(ts => ts.TaskId);
+            modelBuilder.Entity<TaskAndSkill>()
+                .HasOne(s => s.Skill)
+                .WithMany(ts => ts.TaskAndSkills)
+                .HasForeignKey(ts => ts.SkillId);
```

Comment:
This should not be needed to add. You just need to fix ForeignKeys attributes in the TaskAndSkill entity.